tmt/steps/__init__.py

```diff
@@ -783,7 +784,7 @@ def _patch_raw_datum(
             pruned_raw_data: List[_RawStepData] = []
             incoming_raw_datum = _to_raw_step_datum(invocation.options)
 
-            how = invocation.options['how']
+            how = invocation.options.get('how')
```

Comment:
This deserves a comment: how come `how` may be missing in options? Options come from Click, which populates them with default values, which makes `how` present even when not specified on the command line. If this assumption is not true, leave a note here, please.

Reply:
Explained in 02ae2aea.